src/redux/actions/cellSets/runCellSetsClustering.js

```diff
@@ -43,7 +43,7 @@ const runCellSetsClustering = (experimentId, resolution) => async (dispatch, get
           throw err;
         }
 
-        dispatch(updateCellSetsClustering(experimentId));
```

Comment:
Looking [updateCellSetsClustering()](https://github.com/hms-dbmi-cellenics/ui/blob/1e8b7c97357398b1f1a6f0e928631c2a907800e7/src/redux/actions/cellSets/updateCellSetsClustering.js#L7), it seems you still need the experimentId.